### ISSUE TITLE
zt-zip fails to unpack some jar

### ISSUE THREAD

**Issue by slandelle** [zeroturnaround/zt-zip#144]
Hello,

When trying to unpack [this jar](https://repo1.maven.org/maven2/org/scalactic/scalactic_2.13/3.2.5/scalactic_2.13-3.2.5.jar), zt-zip crashes with the following stacktrace:

```
Caused by: org.zeroturnaround.zip.ZipException: java.util.zip.ZipException: bad extra field starting at 9.  Block length of 49135 bytes exceeds remaining data of 4 bytes.
    at org.zeroturnaround.zip.ZipEntryUtil.getZTFilePermissions (ZipEntryUtil.java:202)
    at org.zeroturnaround.zip.ZipUtil$Unpacker.process (ZipUtil.java:1182)
    at org.zeroturnaround.zip.ZipUtil.iterate (ZipUtil.java:508)
    at org.zeroturnaround.zip.ZipUtil.iterate (ZipUtil.java:472)
    at org.zeroturnaround.zip.ZipUtil.unpack (ZipUtil.java:996)
```

I have no problem unzipping this file with any desktop tool, or if I implement the unzip myself based on `ZipInputStream`.

**Comment by toomasr** [comment]
The JAR file has problems. When I run `unzip -t scalactic_2.13-3.2.5.jar` on the downloaded file then there are 305 errors in the JAR file. It does unzip though. I wonder what is a good solution here. Able to ignore errors maybe?

**Comment by slandelle** [comment]
Wow, you're right.

> Able to ignore errors maybe?

That would be great.
IMHO that should be the default behavior to match default unzip or the JVM that doesn't have any issue loading this jar.
And maybe add an option to have the strict existing behavior.

**Comment by toomasr** [comment]
I took a look and actually we should ignore the error. I'll see if I still have permissions to do a maven release later today.

**Comment by toomasr** [comment]
I've released 1.15. Your issue should be fixed now.

**Comment by slandelle** [comment]
Thanks a lot!
